source/helpers/attach-element.ts

```diff
@@ -23,7 +23,12 @@ Get unique ID by using the line:column of the call (or its parents) as seed. Eve
 @param ancestor Which call in the stack should be used as key. 0 means the exact line where getSnapshotUUID is called. Defaults to 1 because it's usually used inside a helper.
 */
 export function getSnapshotUUID(ancestor = 1): string {
-	return hashString(new Error('Get stack').stack!.split('\n')[ancestor + 2]);
+	const stack = new Error('Get stack').stack!.split('\n');
+	if (stack[0] === 'Error: Get stack') {
+		stack.splice(0, 1);
```

Comment:
This should be unshift() I think. Also it would benefit from a comment explaining this condition, ending it with `#6032`

Reply:
It should be [`shift`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift), not [`unshift`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift):

```suggestion
		stack.shift(); // Remove non-stacktrace line from array (only present in Chrome) #6032
```